src/wfcomman.c

```diff
@@ -714,7 +714,7 @@ OpenOrEditSelection(HWND hwndActive, BOOL fEdit)
       }
       else
       {
-          ret = ExecProgram(szPath, szNULL, NULL, (GetKeyState(VK_SHIFT) < 0), FALSE);
+          ret = ExecProgram(szPath, szNULL, NULL, (GetKeyState(VK_SHIFT) < 0), (GetKeyState(VK_CONTROL) < 0));

```

Comment:
I assume you decided there's no point having this check in the notepad/editor/F12 cases above? "Edit file as admin" seems more useful IMHO to "open the file in a minimized editor".

Reply:
I didn't need 'Edit as admin'.
I needed to run bat files as admin, the same way as Explorer does with 'Run As Administrator'.

But you are right we could also do this for fEdit == TRUE, to have it 'symmetrical'